hatch.toml

```diff
@@ -9,3 +9,11 @@ dependencies = ["setuptools"]  # https://bitbucket.org/pybtex-devs/pybtex/issues
 [envs.docs.scripts]
 build = "sphinx-build -M html docs docs/_build -W --keep-going {args}"
 clean = "git clean -fX -- docs"
+
+[envs.towncrier]
+features = ["doc"]
+dependencies = ["setuptools"]  # https://bitbucket.org/pybtex-devs/pybtex/issues/169
```

Comment:
Subject to change upon #1625

Reply:
what are you referring to? the `doc` feature will do what we need, no?

Reply:
Using https://github.com/scverse/anndata/pull/1625/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R86 instead i.e., `dev-doc`.  We don't need all of `doc`, no?

Reply:
true, and that’s merged now.

you can just use the default env, already has towncrier in it.